docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+    </recordInfo>
+
+recordCreationDate
+------------------
+
+Use Case
+^^^^^^^^
+
+A total of 167 values are present for <recordCreationDate>. This value shows when the record was originally created. All
+but one of these values precedes 2010. All of the recently migrated TEI SCOUT records (2,386) have a value of
+"2020-04-23-04:00". This is the only value not presented in EDTF format. Otherwise all of the values appear to come from
+Volunteer Voices.
+
+Justification
+^^^^^^^^^^^^^
+
+Unlike <recordChangeDate>, all of the values within <recordCreationDate> are at least accurate. Currently this information
+is not used or displayed for users. Given this and the fact that this element is present in a very small percentage of
+UTK records, it does not seem useful to keep this information. Again, a repository system should have a way to track
+when a metadata datastream for a particular digital object was created. Therefore keeping this information adds unnecessary
+complexity.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordCreationDate"
+
+Decision
+^^^^^^^^
+
+These values will be dropped.
+
+Here's an `example record - volvoices:1857 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A1857/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordCreationDate encoding="edtf">2007-10-26</recordCreationDate>
+    </recordInfo>
+
+recordContentSource
+-------------------
+
+Use Case
+^^^^^^^^
+
+The <recordContentSource> element is one of the most essential elements within <recordInfo>, as we currently use it to
+communicate the provider in DPLA. At UTK though, the information we share in this element is not consistent with the
+definition of <recordContentSource> - "The code or name of the entity (e.g. an organization and/or database) that either
+created or modified the original record." While we work with other partners, like the Children's Defense Fund and the
+McClung Museum, we are still technically the creators of the records in these situations. Despite this, we typically list
+these institutions as the record creator because we set up <recordContentSource> as the element that DPLA should map to
+for content provider. In actuality, when the content provider is not UTK, this information should be communicated in
+<physicalLocation> and our DPLA mapping should be updated. Despite these semantic issues, UTK has consistently put this
+information in an incorrect element, so the mapping is not affected. In actuality then, the mapping below is for the
+content provider or institution that would be present in <physicalLocation> if MODS was followed more closely.
+
+Justification
+^^^^^^^^^^^^^
+
+A content provider is required in DPLA. This value also provides UTK with the opportunity to attribute collections to
+the institution that provided them, which is important for maintaining respectful relationships.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordContentSource"
```

Comment:
Normally, we express an xpath here.  Are you trying to say:

```
mods:recordInfo/mods:recordContentSource
```

Reply:
This was resolved in https://github.com/UTKcataloging/mods_to_rdf/pull/27/commits/2bd47b57b297b1805e1b9a7e6f517ce517eda056